docs/GettingStarted.md

```diff
@@ -36,6 +36,10 @@ The following steps need to be executed _once_ before the first build on a new c
 2. Change directory to where the project is cloned, e.g. ```cd ebpf-for-windows```.
 3. ```msbuild /m /p:Configuration=Debug /p:Platform=x64 ebpf-for-windows.sln```
 
+### Setting compile time options when building from Developer Command Prompt
```

Comment:
```suggestion
#### Setting compile time options when building from Developer Command Prompt
```
this should be a subsection of the previous since as written it's specific to using msbuild from the command line.

But what if they're using the Visual Studio IDE, can we cover that too either in that section, or make this section apply to both and move it down?

Reply:
Unresolving since I didn't see an answer to my second sentence ("But what if they're using the Visual Studio IDE, can we cover that too either in that section, or make this section apply to both and move it down?").  Let me know if I missed it.